with-astro/.npmrc

```diff
@@ -0,0 +1,2 @@
+## force pnpm to hoist 
+shamefully-hoist = true
```

Comment:
I think this file can be removed. We're not using pnpm here, right?

Reply:
Done